src/show/pages/private/CreateProject/Subviews/CategoryStep/index.js

```diff
@@ -32,4 +38,12 @@ const CategoryStep = ({ nextStep, project, setProject }) => {
   );
 };
 
+CategoryStep.defaultProps = {
+  nextStep: () => {}
```

Comment:
And of course, use it in every defaultProps.

Reply:
Even though this looks redundant, it's better because we don't initialise the functions every time we define a new component's default props (and when the component mounts), but we only define it once and reference/call it every where we want to use it.